.tekton/cli-build.yaml

```diff
@@ -52,17 +52,17 @@ spec:
       description: Build dependencies to be prefetched by Cachi2
       name: prefetch-input
       type: string
-    - default: "false"
-      description: Java build
-      name: java
-      type: string
```

Comment:
Looks like this was removed, see https://github.com/konflux-ci/build-definitions/pull/1325